edx_name_affirmation/tests/test_models.py

```diff
@@ -1,15 +1,40 @@
 """
 Tests for Name Affirmation models
 """
+from unittest.mock import patch
 
 from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
 from django.test import TestCase
 
 from edx_name_affirmation.models import VerifiedName
 from edx_name_affirmation.statuses import VerifiedNameStatus
 
 User = get_user_model()
 
+idv_attempt_id = 34455
```

Comment:
Nit. `idv_attempt_id` makes sense to define in a common place because it's used across test methods, but I think making it an attribute of the test class instance, set in a `setUp` method would be better.

```
def setUp(self):
    super().setUp()
    self.idv_attempt_id = 3455
```

and then change references to it in the tests to `self.idv_attempt_id`.

Reply:
`idv_attempt_id` is being used within the helper method `_mocked_model_get()`. I'll try to scope it inside the test. Maybe just adding it as a helper method there is better.

Reply:
I moved these variables and the helper methods to be defined within `VerifiedNameModelTests` class so the scope is more limited. Let me know what you think.